### ISSUE TITLE
keyboard is broken on current git

### ISSUE THREAD

**Issue by ghomem** [FreeRDP/FreeRDP#1127]
Hi,

dfreerdp from freerdp 1.0.1 had a fully working keyboard. 

I upgraded to current git and the following keys are dead:

AltGr
Right Control
All cursors
Insert, Delete, Home, End, PgUp, PgDown
Print Screen
Pause
/ (from the numerical keypad)

What I mean by dead is Xev, when run from inside the RDP session, not reacting to them at all.

I am sure there is no problem on the server, which is xrdp 0.6, because I tested connecting with

dfreerdp 1.0.1
xfreerdp 1.0.1
rdesktop

and keyboard is fine.

In short: between 1.0.1 and today's git something must have broken the keyboard mapping.
